src/screens/EventsScreen/FilterHeader.test.js

```diff
@@ -70,26 +73,9 @@ describe("renders correctly", () => {
       selectedCategories: new Set(),
       onFilterCategoriesPress: () => {},
       onFilterButtonPress: () => {},
+      onDateFilterButtonPress: () => {},
       numTagFiltersSelected: 2
     });
     expect(output).toMatchSnapshot();
   });
 });
-
-it("opens date picker when users presses date filter button", () => {
```

Comment:
Can we add a test to make sure that the correct handler is called. I.e. `onDateFilterButtonPress` is called;

Reply:
If tests are missing for the others, we should add that too.

Reply:
done